src/conduits/RwaOutputConduit3.sol

```diff
@@ -0,0 +1,263 @@
+// Copyright (C) 2020, 2021 Lev Livnev <lev@liv.nev.org.uk>
+// Copyright (C) 2022 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.6.12;
+
+import {DSTokenAbstract} from "dss-interfaces/dapp/DSTokenAbstract.sol";
+import {PsmAbstract} from "dss-interfaces/dss/PsmAbstract.sol";
+import {GemJoinAbstract} from "dss-interfaces/dss/GemJoinAbstract.sol";
+
+/**
+ * @author Lev Livnev <lev@liv.nev.org.uk>
+ * @author Kaue Cano <kaue@clio.finance>
+ * @author Henrique Barcelos <henrique@clio.finance>
```

Comment:
```suggestion
```

Since only you worked in this file, I think you can remove Kaue and me.

Reply:
Done